### PR TITLE
Fix virtual_disks gluster default_pool cases failures

### PR DESCRIPTION
**Pull request by chunfuwen**
Previous logic will remove VM system disk, which leads to login timeout in VM itself
So remove these logic

Signed-off-by: chunfuwen <chwen@redhat.com>

### CODE REVIEW

**Comment by chunfuwen** [comment]
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.gluster.hot_plug.disk_raw.attach-device.default_pool

File: libvirt/tests/src/virtual_disks/virtual_disks_gluster.py [hunk 1]
```diff
@@ -203,9 +203,6 @@ def test_pmsuspend(vm_name):
                 logging.debug("Can't convert '%s' to integer type",
                               dom_iothreads)
         if default_pool:
-            disks_dev = vmxml.get_devices(device_type="disk")
-            for disk in disks_dev:
-                vmxml.del_device(disk)
             vmxml.sync()
```
Comment (kylazhang) [inline libvirt/tests/src/virtual_disks/virtual_disks_gluster.py:206]: If no more operation for vmxml, then no need for branch and sync?

Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_gluster.py:206]: not sure whether there is some vmxml updates in previous code logic, I assume there is no harm to keep them here

Comment (kylazhang) [reply libvirt/tests/src/virtual_disks/virtual_disks_gluster.py:206]: From my check, no usage as only simple several vmxml operation. We do not suggest to reserve useless code. You can check again and make sure.

**Comment by chunfuwen** [comment]
retest after all code removed:
cleaning libvirtd logs...
DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
PASS 1-type_specific.io-github-autotest-libvirt.virtual_disks.gluster.hot_plug.disk_raw.attach-device.default_pool

File: libvirt/tests/src/virtual_disks/virtual_disks_gluster.py (hunk 1 above)
Comment (chunfuwen) [reply libvirt/tests/src/virtual_disks/virtual_disks_gluster.py]: removed and retest, see updated result

**Review by kylazhang — APPROVED** [review]

**Comment by chunfuwen** [comment]
this is just remove unused 4 lines code, mark it as minor_fix

**Pull request merged**